pkg/storage/driver/mock_test.go

```diff
@@ -142,3 +142,81 @@ func (mock *MockConfigMapsInterface) Delete(name string, opts *metav1.DeleteOpti
 	delete(mock.objects, name)
 	return nil
 }
+
+// newTestFixture initializes a MockSecretsInterface.
+// Secrets are created for each release provided.
+func newTestFixtureSecrets(t *testing.T, releases ...*rspb.Release) *Secrets {
+	var mock MockSecretsInterface
+	mock.Init(t, releases...)
+
+	return NewSecrets(&mock)
+}
+
+// MockSecretsInterface mocks a kubernetes SecretsInterface
+type MockSecretsInterface struct {
+	internalversion.SecretInterface
+
+	objects map[string]*api.Secret
+}
+
+// Init initializes the MockSecretsInterface with the set of releases.
+func (mock *MockSecretsInterface) Init(t *testing.T, releases ...*rspb.Release) {
+	mock.objects = map[string]*api.Secret{}
+
+	for _, rls := range releases {
+		objkey := testKey(rls.Name, rls.Version)
+
+		secret, err := newSecretsObject(objkey, rls, nil)
+		if err != nil {
+			t.Fatalf("Failed to create configmap: %s", err)
+		}
+		mock.objects[objkey] = secret
+	}
+}
+
+// Get returns the Secret by name.
+func (mock *MockSecretsInterface) Get(name string, options metav1.GetOptions) (*api.Secret, error) {
+	object, ok := mock.objects[name]
+	if !ok {
+		return nil, apierrors.NewNotFound(api.Resource("tests"), name)
+	}
+	return object, nil
+}
+
+// List returns the a of Secret.
```

Comment:
Nit: Comment looks like it has a copy/paste error

Reply:
Looks like line 108 of the same file has the same typo. Not entirely sure what the word should be, probably `list` ?

Reply:
Yeah I think so. Some thing like "returns a list of secrets"